app/policies/facility_policy.rb

```diff
@@ -31,6 +31,10 @@ def graphics?
     show?
   end
 
+  def upload?
+    user.owner?
```

Comment:
Won't this be the same as facility_group_policy?

Reply:
Removing the facility_group_policy for upload?, since I'm not currently using it.